### DISCUSSION TITLE
Downstream analysis examples

### DISCUSSION THREAD

**Discussion by jadolfbr** [brianhie/evolocity#9] [General]
I was able to run the code through on a sample set of homologs.  Thanks for the fixes to get this rolling!
I followed the tutorial, which was nice, but I was wondering if you had any example of some downstream analysis you have in a figure in your paper.  You mention inferring roots, ordering in psuedotime, and identifying mutations.  Have you tried any of this and have any examples of using the output data for this?

**Comment by brianhie** [comment]
Here is a Colab tutorial on doing root estimation and pseudotime ordering for influenza NP: https://colab.research.google.com/drive/143vxIqgIO1bWbC1FJ0q_Ja8eZN603uwM?usp=sharing

The paper also has examples of identifying mutations (code for that is here: https://github.com/brianhie/evolocity/blob/3ed6fe076a1b6004cda587a5db732becb6742ae0/bin/flu_np.py#L532, though it's not yet in a nice Colab form)

**Comment by jadolfbr** [comment]
This is awesome!  Thank you!
